src/applications/vre/28-8832/config/chapters/veteran-information/veteranAddress.js

```diff
@@ -0,0 +1,45 @@
+import emailUI from 'platform/forms-system/src/js/definitions/email';
+import {
+  buildAddressSchema,
+  addressUISchema,
+} from '../../../../../disability-benefits/686c-674/config/address-schema';
+import { isVeteran } from '../../helpers';
+
+const veteranAddress = buildAddressSchema(true);
+// reset boolean type for checkbox
+veteranAddress.properties['view:livesOnMilitaryBase'] = {
+  type: 'boolean',
+};
+
+export const schema = {
+  type: 'object',
+  properties: {
+    veteranAddress,
+    phone: {
+      type: 'string',
+      minLength: 10,
+    },
+    emailAddress: {
+      type: 'string',
+      minLength: 6,
+      maxLength: 80,
+      pattern:
+        '^[_A-Za-z0-9-]+(\\.[_A-Za-z0-9-]+)*@[A-Za-z0-9-]+(\\.[A-Za-z0-9]+)*(\\.[A-Za-z]{2,})$',
+    },
+  },
+};
+
+export const uiSchema = {
+  veteranAddress: addressUISchema(true, 'veteranAddress', () => true),
```

Comment:
Same comment as the dependent address. I think the required callback needs to be based on the status of the user. Could `isVeteran()` be used here?

Reply:
@cohnjesse - did you see my above comment? I think you should be able to pass in a callback such as `isVeteran` since it's just getting assigned to `ui:required` which has formData in its scope.

Reply:
@micahchiang Nope, I tried it and it doesn't work because when you pass it into `veteranAddress:` that `formData` variable comes back as undefined.